astropy/utils/iers/iers.py

```diff
@@ -399,8 +403,7 @@ class IERS_A(IERS):
     Notes
     -----
     The IERS A file is not part of astropy.  It can be downloaded from
-    ``iers.IERS_A_URL``.  See ``iers.__doc__`` for instructions on how to use
-    it in ``Time``, etc.
+    ``iers.IERS_A_URL`` or ``iers.IERS_A_URL_MIRROR``. See ``iers.__doc__`` for instructions on use in ``Time``, etc.
```

Comment:
Oh, no, now it really is too long! I just didn't want to add an extra line; just break after ``iers.__doct__``